src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const rows = [...tbody.rows];
+
+thead.addEventListener('click', (e) => {
+  if (e.target.tagName !== 'TH') {
+    return;
+  }
+
+  const targetIndex = e.target.closest('th').cellIndex;
+
+  const result = rows.sort((a, b) => {
+    const firstCell = a
+      .querySelectorAll('td')[targetIndex]
+      .innerText.replace(/[$,]/g, '');
+    const secondCell = b
+      .querySelectorAll('td')[targetIndex]
+      .innerText.replace(/[$,]/g, '');
```

Comment:
move a way to get a sell to an own functions since you've used the same code twice